slack/src/main/java/org/togglz/slack/SlackStateRepository.java

```diff
@@ -0,0 +1,62 @@
+package org.togglz.slack;
+
+import org.togglz.core.Feature;
+import org.togglz.core.logging.Log;
+import org.togglz.core.logging.LogFactory;
+import org.togglz.core.repository.FeatureState;
+import org.togglz.core.repository.StateRepository;
+import org.togglz.core.user.UserProvider;
+import org.togglz.slack.config.NotificationConfiguration;
+import org.togglz.slack.notification.Notification;
+import org.togglz.slack.notification.NotificationComposer;
+import org.togglz.slack.sender.AsyncNotifier;
+import org.togglz.slack.sender.NotificationSender;
+import org.togglz.slack.sender.Notifier;
+
+import java.util.List;
+
+public class SlackStateRepository implements StateRepository {
```

Comment:
It's the main entry point to that feature. You could probably add a simple JavaDoc containing among others `@since` and `@author` tags.